README.md

```diff
@@ -23,3 +23,40 @@ gradle bootRun --refresh-dependencies
+| IEXEC_WORKER_TEE_COMPUTE_MAX_HEAP_SIZE_GB     | Max heap size for TEE apps.                                                                                                      | Positive integer | 8                                                         |
+| IEXEC_WORKER_DOCKER_NETWORK_NAME              | Name of the network the LAS is in. Useless if worker runs only standard tasks.                                                   | String           | iexec-worker-net                                          |
+| IEXEC_WORKER_DOCKER_REGISTRY_USERNAME_0       | Username to pull apps from [docker official website](https://hub.docker.com/)                                                    | String           |                                                           |
+| IEXEC_WORKER_DOCKER_REGISTRY_PASSWORD_0       | Password to  pull apps from [docker official website](https://hub.docker.com/)                                                   | String           |                                                           |
+| IEXEC_WORKER_DOCKER_REGISTRY_ADDRESS_1        | Secondary registry address.                                                                                                      | String           |                                                           |
+| IEXEC_WORKER_DOCKER_REGISTRY_USERNAME_1       | Username to pull apps from secondary registry.                                                                                   | String           |                                                           |
+| IEXEC_WORKER_DOCKER_REGISTRY_PASSWORD_1       | Password to pull apps from secondary registry.                                                                                   | String           |                                                           |
+| IEXEC_WORKER_WALLET_PATH                      | Path to the wallet that should be used by the worker.                                                                            | String           | ./src/main/resources/wallet/encrypted-wallet_worker1.json |
+| IEXEC_WORKER_WALLET_PASSWORD                  | Password of the wallet that should be used by the worker.                                                                        | String           | whatever                                                  |
+| IEXEC_SCONE_SHOW_VERSION                      | Whether to display version of Scone.                                                                                             | Boolean          | true                                                      |
+| IEXEC_SCONE_LOG_LEVEL                         | Log level Scone should use.                                                                                                      | [0,7] or String  | debug                                                     |
```

Comment:
With a production mindset, wouldn't it be better to have a default value like `WARN` ?
The fact to have those logs at the `DEBUG` level should be the exception.

Reply:
That's an eternal debate: whether we want to have production-ready code or we want to force users to customize their workers.

Reply:
Yes, we need to work on that someday : `Dev configuration vs Production configuration`, we can add that to the backlog.